docs/index.md

```diff
@@ -428,6 +428,14 @@ For example:
 
 When reading a QR code, scanning software can recognize a SMART Health Card from the `shc:/` prefix. Stripping this prefix and the following `<ordinal>/<count>/` and decoding the remaining pairs of numerals yields a JWS.
 
+## Expiration of Health Cards
+
+Normally, Health Cards do not expire.  Health Cards are usually intended to contain factual information that is assured to be correct at the point of issuance and should not change with the passage of time.  There are, however, situations where the ability to set an expiration date is beneficial.  To be clear, these are special situations and expirations are **NOT** expected to be routinely used when clinical data Health Cards are issued by an authoritative source.
```

Comment:
Ah ok, then I would suggest adding something like:

```suggestion
SMART Health Cards contain factual information that is assured to be correct at the point of issuance and does not change with the passage of time. Therefore, **Health Cards generally do not expire** and an expiration date is not used. There are, however, situations where the ability to set an expiration date is beneficial.
```

Reply:
I'm good with this rephrasing proposed by @p2-apple if @jmandel is OK with it.